broker/openshift-origin-broker.spec

```diff
@@ -79,6 +79,7 @@ Requires:      %{?scl:%scl_prefix}rubygem-metaclass
 Requires:      %{?scl:%scl_prefix}rubygem-mime-types
 Requires:      %{?scl:%scl_prefix}rubygem-moped
 Requires:      %{?scl:%scl_prefix}rubygem-multi_json
+Requires:      %{?scl:%scl_prefix}rubygem-net-ssh
```

Comment:
I'm really sorry, I meant to the controller's RPM specfile. I copied and pasted the wrong thing. This is in addition to the changes I made in the PR against your branch.